### PR TITLE
FIXED : Display a message when there is no activity matching the search in listview

### PR DESCRIPTION
**Pull request by hharshas**

### CODE REVIEW

File: v2/js/screens/firstscreen.js [hunk 1]
```diff
@@ -10,10 +10,10 @@ const FirstScreen = {
...
 									id="help-icon"
 									svgfile="./icons/help.svg"
-									size="50"
-									x="0"
-									y="0"
-									color="256"
+									:size="50"
+									:x="0"
+									:y="0"
+									:color="256"
```
Comment (llaske) [inline v2/js/screens/firstscreen.js:16]: No need to be change.

**Review by llaske — CHANGES_REQUESTED** [review]
Lot of code change is not related to the fix: the add of prefix ":".
Please refresh your branch to limit code change to update need for the fix.

File: v2/js/screens/firstscreen.js (hunk 1 above)
Comment (hharshas) [reply v2/js/screens/firstscreen.js:16]: ohk, resolving these.

**Review by hharshas — COMMENTED** [review]
unnecessary changes have been removed.

File: v2/js/screens/listview.js [hunk 2]
```diff
@@ -41,12 +41,32 @@ const ListView = {
...
+					<div class="no-matching-activities" v-if="activities.length === 0">
+						<div>
+							<icon 
+								svgfile="./icons/entry-search.svg"
+								:color="768"
+								:size="44"
+								isNative="true"
+							></icon>
+						</div>
+						<div> No matching activities </div>
```
Comment (llaske) [inline v2/js/screens/listview.js:53]: Text must be localized.

**Comment by hharshas** [comment]
Can you review this again ?

**Comment by llaske** [comment]
> @llaske Can you review this again ?

Look like nothing has changed since my previous review.

**Comment by hharshas** [comment]
@llaske I localized the texts as you told. Is there something else that I am missing and I have to do ?

**Comment by llaske** [comment]
> @llaske I localized the texts as you told me to do. Is there something else that I am missing and I have to do ?

Yes, see comments on the review https://github.com/llaske/sugarizer/pull/1561/files/c87feda844d05640f652382b1d7bbc651addb2d7

**Comment by hharshas** [comment]
@llaske  I am moving offtrack and unable to understand what to do so here is the summary of all the changes that I have brought.
Motive : Display a message when there is no activity matching the search in listview.
Further Changes  :-
- [x] fix: the add of prefix ":". https://github.com/llaske/sugarizer/pull/1561#pullrequestreview-1937734840 (remove ':')
- [x] Text must be localized. https://github.com/llaske/sugarizer/pull/1561#discussion_r1526401023 (add Sugar.l10n)

Total 6 Files, 60 lines Changed in 9 commits 
Please comment below for further assistant...

**Review by llaske — CHANGES_REQUESTED** [review]
Sorry @hharshas my review was pending. See my comments above.

**Comment by hharshas** [comment]
@llaske  Ohk will resolve these...

**Pull request closed without merging**